bundles/CoreBundle/Command/Targeting/MigrateElementNamesCommand.php

```diff
@@ -107,8 +108,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $this->db = \Pimcore::getContainer()->get(Db\ConnectionInterface::class);
```

Comment:
This is intentional, because if the DB connection isn't set up (e.g. before running the installer) this change will lead to errors (e.g. when running clear:cache). 
But maybe we just change it to `Pimcore\Db::get()` which is shorter and does the same :)

Reply:
not sure, but does it also happen when you lazy load the command? https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading

Reply:
Unfortunately yes. At least last time i tried it.

Reply:
alright. Oro Platform has similar issues, they fixed it by wrapping and catching things inside a special service: https://github.com/oroinc/platform/blob/master/src/Oro/Bundle/EntityBundle/Tools/SafeDatabaseChecker.php

That might work for these use-cases, I am just afraid people might use it in wrong scenarios as well. And: for now Pimcore::getDb might really be the best solution here.

Reply:
@dpfaffenbauer thanks for your feedback! 
@blankse can you please update the PR accordingly? 

Thanks a lot guys!

Reply:
I updated the PR